AuxTel/AuxTel-Non-standard-Operations/AuxTel-EUI-Access/AuxTel-EUI-Access.rst

```diff
@@ -0,0 +1,96 @@
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: Kshitija Kelkar
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: OS Team
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Non-Standard-Operations-AuxTel-EUI-Access:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. role::  raw-html(raw)
+    :format: html
+
+####################
+AuxTel EUI Access
+####################
+
+.. _Informative-Document-Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the document's purpose and utilization. 
+
+.. _Informative-Document-Main-Information-Section:
+
+This document explains how to to access all the components of *AuxTel EUI*-- for ATMCS, ATSPectrograph and ATDome-- through a remote desktop.
+
+AuxTel EUI Access
+========================
+ 
+.. _Auxtel-EUI-Access-Setup:
+
+Setup to access the AuxTel EUI remote desktop 
+---------------------------------------------
+
+#. Install and open **Microsoft Remote Desktop**.
+
+#. Click "+" button on the top menu.
+
+#. Select "Add PC" from drop-down menu.
+
+#. Click the drop-down menu "Gateway" :raw-html:`&rarr;` "Add a Gateway"
+
+.. figure:: /AuxTel/AuxTel-Non-standard-Operations/_static/image-2023-11-8_17-6-47.png
+  :name: "Add PC" pop-up window
+
+
+#. Put *aux-brick01.cp.lsst.org* on the "Gateway name" :raw-html:`&rarr;` "Add"
+
+.. figure:: /AuxTel/AuxTel-Non-standard-Operations/_static/image-2023-11-8_17-3-7.png
+  :name: "Add a Gateway" pop-up window
+
+
+.. note::
+    You need to configure this setup only once. 
+
+.. _Auxtel-EUI-Access-Connection:
+
+Connecting to the EUI Remote desktop
+------------------------------------
+
+#. Login with Username/PW of **ATMCS/ATSpectrograph/ATDome EUI access** on 1Password **AuxTel** Vault.
+
+
+#. The following tabs can now be accesible from the web-browser that should always be open -
```

Comment:
Could you add the addresses in case they are not open?

Reply:
I have decided to not hyperlink them since they are supposed to open and in remote desktop